### DISCUSSION TITLE
Please sanity check the results

### DISCUSSION THREAD

**Discussion by LorenPechtel** [mifi/lossless-cut#1453] [Ideas]
I've been using this on a lot of files and I hit one that finished at about 1%--but there was no error.  Merging files should produce a file of about the size of the sum of the inputs and any substantial deviation from this almost certainly means something went wrong.  Please say so when this happens!

**Comment by mifi** [comment]
What do you mean finishd at about 1%? 1% of the size of the input files? Are you talking about cut+merge file now, or merge separate files into one?

**Comment by LorenPechtel** [comment]
It behaved just like it would when done, but the display only counted up to 1% and the output file size was consistent with that.
________________________________

**Reply by mifi** [reply]
Ok then it’s probably a bug in ffmpeg. but i’ll add a sanity check after merge, so it will show a warning if the output size differs by more than 5% from total input size

**Comment by LorenPechtel** [comment]
Thank you.  It caught one that failed at about 50% so it wasn't so obvious.  (I am keeping the originals around for now, though, just in case.)
________________________________
